src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -19,18 +19,15 @@ public class Arm extends SubsystemBase {
 
   private final CANcoder armLeftEncoderAbs = new CANcoder(RobotMap.ARM_ABS_ENCODER_CAN_ID);
   // trapezoidal motion profiling to account for large jumps in velocity which result in large error
-  private TrapezoidProfile.Constraints trapezoidProfileConstraints = new TrapezoidProfile.Constraints(
-    ArmCal.ARM_MOTOR_MAX_VELOCITY_RPS, ArmCal.ARM_MOTOR_MAX_ACCERLATION_RPS_SQUARED);
   private final TrapezoidProfile trapezoidProfile =
-      new TrapezoidProfile(trapezoidProfileConstraints);
+      new TrapezoidProfile(new TrapezoidProfile.Constraints(
```

Comment:
So, I now see that Sid took out the TrapezoidProfile.Constraints and made them their own instance variables so he could add the values to ShuffleBoard and use their setters ... but I still don’t we need them and they were implemented incorrectly anyway